int8 n_register
int8 bit
int8 value
---
bool ret